### PR TITLE
Consumption Report for Aura VDC customers

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by fiquick** [comment]
[Consumption Report for Aura VDC customers](https://trello.com/c/zqkFyaG4/359-consumption-report-for-aura-vdc-customers)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-470.surge.sh

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-470.surge.sh

New pages:
- [Consumption report](https://neo4j-docs-aura-470.surge.sh/aura/platform/consumption-report)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-470.surge.sh

New pages:
- [Consumption report](https://neo4j-docs-aura-470.surge.sh/aura/platform/consumption-report)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-470.surge.sh

New pages:
- [Consumption report](https://neo4j-docs-aura-470.surge.sh/aura/platform/consumption-report)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-470.surge.sh

New pages:
- [Consumption report](https://neo4j-docs-aura-470.surge.sh/aura/platform/consumption-report)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-470.surge.sh

New pages:
- [Consumption report](https://neo4j-docs-aura-470.surge.sh/aura/platform/consumption-report)

File: modules/ROOT/pages/platform/consumption-report.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[[aura-consumption-report]]
+= Consumption report
+:description: Consumption reporting allows Virtual Dedicated Cloud customers to closely monitor their consumption.
+
+label:AuraDB-Virtual-Dedicated-Cloud[]
+
+Virtual Dedicated Cloud services are offered through prepaid consumption plans. Billing is based on usage, with credits deducted from the available balance each month.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/consumption-report.adoc:7]:
```suggestion
Virtual Dedicated Cloud services are offered through prepaid consumption plans. 
Billing is based on usage, with credits deducted from the available balance each month.
```

File: modules/ROOT/pages/platform/consumption-report.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+The consumption report, accessible in the Aura console’s Billing section, provides real-time insights into resource usage for the current tenant, including both running and paused states. 
+It displays RAM usage in GB-hours and the equivalent cost in prepaid credits.
+
+Available to Admins, the report helps you track usage patterns over time and make informed resource allocation decisions. 
+Note that it includes primary database usage but excludes secondary database usage.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/consumption-report.adoc:13]:
```suggestion
Note that it includes primary database usage but **not** secondary database usage.
```

File: modules/ROOT/pages/platform/consumption-report.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+== Monitor consumption in real-time
+
+=== Billing category
+
+The consumption report shows the billing status, which can be `running` meaning customers are charged the full price, or `ended` meaning customers pay 20% of the hourly rate.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/consumption-report.adoc:19]:
```suggestion
The consumption report shows the billing status, which can be `running` meaning customers are charged the full price, or `ended` meaning customers are charged 20% of the hourly rate.
```

**Review by AlexicaWright — COMMENTED** [review]
Some suggestions, but overall it looks good!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-470.surge.sh

New pages:
- [Consumption report](https://neo4j-docs-aura-470.surge.sh/aura/platform/consumption-report)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-470.surge.sh

New pages:
- [Consumption report](https://neo4j-docs-aura-470.surge.sh/aura/platform/consumption-report)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-470.surge.sh

New pages:
- [Consumption report](https://neo4j-docs-aura-470.surge.sh/aura/platform/consumption-report)

File: modules/ROOT/pages/platform/consumption-report.adoc [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+label:AuraDB-Virtual-Dedicated-Cloud[]
+
+Virtual Dedicated Cloud services are offered through prepaid consumption plans. 
+Billing is based on usage, with credits deducted from the available balance each month.
+
+The consumption report, accessible in the Aura console’s Billing section, provides real-time insights into resource usage for the current tenant, including both running and paused states. 
+It displays RAM usage in GB-hours and the equivalent cost in prepaid credits.
+
+Available to Admins, the report helps you track usage patterns over time and make informed resource allocation decisions. 
+Note that it includes primary database usage but **not** secondary database usage.
```
Comment (basturzoiu) [inline modules/ROOT/pages/platform/consumption-report.adoc:14]: note that the "secondary" external-facing name might be different.

File: modules/ROOT/pages/platform/consumption-report.adoc [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+== Monitor consumption in real-time
+
+=== Billing category
+
+The consumption report shows the billing status, which can be `running` meaning customers are charged the full price, or `ended` meaning customers are charged 20% of the hourly rate.
```
Comment (basturzoiu) [inline modules/ROOT/pages/platform/consumption-report.adoc:20]: `paused`

File: modules/ROOT/pages/platform/consumption-report.adoc (hunk 4 above)
Comment (fiquick) [reply modules/ROOT/pages/platform/consumption-report.adoc:14]: OK! We have a draft PR here https://github.com/neo4j/docs-aura/pull/396

File: modules/ROOT/pages/platform/consumption-report.adoc (hunk 5 above)
Comment (fiquick) [reply modules/ROOT/pages/platform/consumption-report.adoc]: thanks, updated!

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-470.surge.sh

New pages:
- [Consumption report](https://neo4j-docs-aura-470.surge.sh/aura/platform/consumption-report)

**Pull request merged**